AppCenterCrashes/AppCenterCrashes/MSACCrashes.h

```diff
@@ -148,16 +149,14 @@ typedef NS_ENUM(NSUInteger, MSACUserConfirmation) {
  *
  * @see MSACCrashesDelegate
  */
-+ (void)setDelegate:(_Nullable id<MSACCrashesDelegate>)delegate;
+@property(class, nonatomic) id<MSACCrashesDelegate> _Nullable delegate;
 
 /**
  * Set a user confirmation handler that is invoked right before processing crash reports to determine whether sending crash reports or not.
  *
- * @param userConfirmationHandler A handler for user confirmation.
- *
  * @see MSACUserConfirmationHandler
  */
-+ (void)setUserConfirmationHandler:(_Nullable MSACUserConfirmationHandler)userConfirmationHandler;
+@property(class, nonatomic) MSACUserConfirmationHandler _Nullable userConfirmationHandler;
```

Comment:
BTW should we swiftify `MSACUserConfirmationHandler`, it still has the `MSAC` prefix in Swift?

Reply:
> The public delegates (e.g. `MSACCrashesDelegate`) have more complicated API names I feel like we should do a pass on them. Particularly paying attention to [omit words that merely repeat type information](https://swift.org/documentation/api-design-guidelines/).
> Other than that I have seen all the files I wanted to review :).

Regarding the delegates, I think swift api already looks fine. Eg:
```
- (void)crashes:(MSACCrashes *)crashes didFailSendingErrorReport:(MSACErrorReport *)errorReport withError:(NSError *)error;
```

Resolves to swift as
```
func crashes(_ crashes: Crashes, didFailSending errorReport: ErrorReport, withError error: Error) {
}
```

Here is the place in swift code https://github.com/microsoft/appcenter-sdk-apple/blob/develop/Sasquatch/SasquatchSwift/AppDelegate.swift#L217

Do you think there still should be some additional swiftify changes applied?